dartagnan/src/main/java/com/dat3m/dartagnan/wmm/definition/ReadModifyWrites.java

```diff
@@ -0,0 +1,20 @@
+package com.dat3m.dartagnan.wmm.definition;
+
+import com.dat3m.dartagnan.wmm.Definition;
+import com.dat3m.dartagnan.wmm.Relation;
+
+/*
+    NOTE: Changes to the semantics of this class may need to be reflected
+    in RMWGraph for Refinement!
+ */
```

Comment:
This class has not semantics information anymore. The comment can go

Reply:
Better move it to `RelationAnalysis`